src/community/long-scale.ts

```diff
@@ -0,0 +1,24 @@
+import { Notation } from "../notation";
+import type Decimal from "break_infinity.js";
+import { toFixedLongScale, abbreviate } from "../utils";
+
+// Name comes from https://en.wikipedia.org/wiki/Long_and_short_scales
+// Period-comma swaps are because where the long scale is used, the period
+// is generally used rather than the comma in numbers compared to where
+// the short scale is used, and vice versa.
+export class LongScaleNotation extends Notation {
+  public readonly name = "Long scale";
```

Comment:
Codacy found an issue: [expected member-variable-declaration: 'name' to have a typedef](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6218257)